examples/templateservicebroker/apiserver-config.yaml

```diff
@@ -0,0 +1,4 @@
+kind: TemplateServiceBrokerConfig
+apiVersion: config.templateservicebroker.openshift.io/v1
+templateNamespaces:
+- openshift
```

Comment:
> wouldn't the ansible installer be using the full TSB template file and customizing/deploying that?

It's because the yaml file provides input to the template in a way that it easier to understand.



> (in the same way that cluster up does?)

If you'd like me to factor out and provide addition options in cluster-up to support ansible-like use-cases I can, but it seem unnecessary.